sources/Core/ImageFlags.cpp

```diff
@@ -526,6 +526,10 @@ static void ConvertImageBufferFormatWorker(
     VariantConstBuffer  srcBuffer = srcImageView.data;
     VariantBuffer       dstBuffer = dstImageView.data;
 
+    const std::size_t y = begin / extent.width;
```

Comment:
Makes sense to me, I didn't think of the previous padding that would have been necessary when the worker starts half way through the buffer. I knew this will need more testing, but I only came up with a quick and simple unit test so far.